lambda-functions/Lambda/SQLAlchemy/db_utils.py

```diff
@@ -0,0 +1,11 @@
+from subprocess import call
+
+# on EC2 instance with AWS CLI configured for my user
+# db_host = call("aws rds describe-db-instances | jq -r '.DBInstances[]|select(.DBInstanceIdentifier=='rds-reddit-data').Endpoint|.Address'", shell=True)
+# else use :
+db_host = "postgresql://hiddenalphabet:hiddenalphabet@rds-reddit-data.cwnallhzqgag.us-west-2.rds.amazonaws.com:5432/hiddenalphabet_reddit_data"
```

Comment:
Make sure to remove these credentials, and access them either via environment variables or loading a config file.